build.sbt

```diff
@@ -1,15 +1,10 @@
 sourceDirectory := file("dummy source directory")
 
-val scalaMajorVersion = SettingKey[Int]("scalaMajorVersion")
-
-scalaVersionSettings
-
 val Scala212 = "2.12.13"
 val Scala213 = "2.13.5"
-val DottyOld = "3.0.0-RC1"
-val DottyNew = "3.0.0-RC2"
+val Scala30 = "3.0.0-RC2"
 
-ThisBuild / crossScalaVersions := Seq(DottyOld, DottyNew, Scala212, Scala213)
+ThisBuild / crossScalaVersions := Seq(Scala30, Scala212, Scala213)
```

Comment:
We need to decide when we want to do this change, because building for anything less than two Dotty versions is going to be extremely painful for downstream libraries (which is virtually everyone). We might have to wait for the next Dotty RC to hit.

Reply:
Ok, I'm out of touch.  I thought this part of the build was an artifact of switching from 0.27 and I assumed things were converging now that Dotty had reached release candidate status.  What's the need for publishing multiple Dotty versions?

Reply:
It's not strictly necessary, but it allows people to bump their ScalaCheck version separately from bumping their Scala 3 version. This can help cut down the number of possible causes to be investigated when something goes wrong.

The old Dotty / Scala 3 versions do rapidly become obsolete, but sometimes a particular one has a bug that's a showstopper for a particular project, and sometimes people are a version behind while they wait for long dependency chains to publish.

Summary: publishing for the last two prereleases isn't sufficiently important to do if it involves real difficulty, but if it's merely a matter of leaving both versions in a matrix, then we might as well.

Reply:
Ok, the convenience for version flexibility make sense. Happy to put back.